package.json

```diff
@@ -181,6 +181,30 @@
   "pnpm": {
     "overrides": {
       "usocket": "_EXCLUDED_"
+    },
+    "peerDependencyRules": {
+      "allowedVersions": {
+        "esbuild-sass-plugin>esbuild": "^0.16.7",
+        "react-electron-web-view>react": "^18.0.0",
+        "react-electron-web-view>react-dom": "^18.0.0",
+        "react-loader>react": "^18.0.0",
+        "react-loader>react-dom": "^18.0.0",
+        "react-sortable-hoc>react": "^18.0.0",
+        "react-sortable-hoc>react-dom": "^18.0.0"
+      }
+    },
+    "allowedDeprecatedVersions": {
+      "kleur": "2.0.2",
+      "set-value": "1.0.0",
+      "request": "2.88.2",
+      "har-validator": "5.1.5",
+      "uuid": "3.4.0",
+      "@sinonjs/fake-timers": "10.2.0",
+      "@npmcli/move-file": "1.1.2 || 2.0.1",
+      "source-map-resolve": "0.5.3",
+      "source-map-url": "0.4.1",
+      "resolve-url": "0.2.1",
+      "urix": "0.1.0"
```

Comment:
These warnings are now silenced, and they should be worked down one by one ideally.

It is possible to trace back which dependencies need to be upgraded by running e.g. `pnpm why uuid`. In that case, `@adonisjs/framework` is the culprit.

Reply:
please add TODO comments in the code - once this PR is closed, we will lose track of such info